src/App.tsx

```diff
@@ -1,26 +1,156 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState, useMemo } from 'react';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import classNames from 'classnames';
+import { USER_ID } from './api/todos';
+import { SelectOption } from './types/Todo';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string | null>('');
+  const [filter, setFilter] = useState(SelectOption.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [inputValue, setInputValue] = useState('');
+  const [todoDelete, setTodoDelete] = useState<number[]>([]);
+  const userId = USER_ID;
+
+  function close() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const close = () => {
```